package.json

```diff
@@ -98,6 +98,7 @@
     "chalk": "^5.2.0",
     "concurrently": "^9.0.0",
     "dpdm": "^3.14.0",
+    "earcut": "^3.0.1",
```

Comment:
> If needed we can "fix" the earcut version so it doesn't automatically update on patch or minor changes.

I would prefer a fixed version so there is no unexpected alteration in a triangulation.

Reply:
Done!